index.md

```diff
@@ -8,21 +7,18 @@ submit digital objects to a repository system; the Archival Information Package
 storage over the long-term; and the Dissemination Information Package (DIP)
 which is used to disseminate digital objects to the requesting user.
 
-In this context, the current document represents the specification of the E-ARK
-Archival Information Package format (E-ARK AIP, in the following named shortly
-AIP). It defines the specific requirements for archiving and storing information
-packages for the long term and focuses on the structural peculiarities and
-metadata requirements relating to the AIP. Considered in detail, the key
-objectives of this format are to:
-
-- define a generic structure of the AIP format in a way that it is suitable for
-  a wide variety of data types, such as document and image collections, archival
-records, databases or geographical data.
-- recommend a set of metadata standards related to the structural and the
+This document is specification of the E-ARK Archival Information Package format
+(E-ARK AIP, in the following named shortly AIP). It defines requirements and
+guidelines for creating AIPs which are adequate to store information packages
+for the long term. The key objectives of this format are to:
+
+- define the AIP format as an extension of the E-ARK CSIP so that it is suitable for
+  the long-term storage of a wide variety of data types, such as document and image
+  collections, archival records, databases or geographical data.
+- recommend specific ways of using metadata standards related to the
   preservation aspects of the AIP.
-- ensure the format is suitable to store large quantities of data.
-- mitigate the potential preservation risk of repository obsolescence by
-  implementing a repository succession strategy.
+- specify a form of packaging AIP container files while ensuring that the
+  format is suitable to store large quantities of data.
```

Comment:
is suitable to store large quantities of data == is suitable for the storage of large quantities of data

Reply:
done